app/helpers/sage/modules_helper.rb

```diff
@@ -1,13 +1,21 @@
 module Sage
   module ModulesHelper
+
+    # Lists out all the available sage modules
     def sage_modules
       [
-        { title: "Assistant", path: "assistant" },
-        { title: "Content", path: "content" },
-        { title: "Nav", path: "nav" },
-        { title: "Page", path: "page" },
-        { title: "Sidebar", path: "sidebar" },
+        { title: "assistant" },
+        { title: "content" },
+        { title: "nav" },
+        { title: "page" },
+        { title: "sidebar" },
       ]
     end
+
+    # Sorts available modules based on alphabet
+    def sorted_sage_modules
+      sage_modules.sort_by { |h| h.first }
```

Comment:
```suggestion
      sage_modules.sort_by { |h| h[:title] }
```
Since the sort element is the full hash, the element ordering matters.
```ruby
> [{title: "b"},{zoo: "animal", title: "a"}].sort_by { |h| h.first }
=> [{:title=>"b"}, {:zoo=>"animal", :title=>"a"}]
```
This is optional since we're defining the hashes, but it could trip someone up if they change the structure of the hash.  If this behavior is what's intended, then disregard this suggestion.